py3status/modules/wifi.py

```diff
@@ -11,9 +11,8 @@
     device: Wireless device name (default "wlan0")
     down_color: Output color when disconnected, possible values:
         "good", "degraded", "bad" (default "bad")
-    format_down: Output when disconnected (default "W: down")
-    format_up: See placeholders below
-        (default "W: {bitrate} {signal_percent} {ssid}")
+    format: Display format for this module
+        (default 'W: [{bitrate} {signal_percent} {ssid}|\?show down]')
```

Comment:
`format = '[W: {bitrate} {signal_percent} {ssid}|\?show W: down]'` 

I added `W`. Will remove `?show` if it's not necessary. Btw, wifi deprecation does not work for me... so that's where we are stuck at right now unless you can confirm that it's working okay.